examples/tests/valid/struct_from_json.test.w

```diff
@@ -262,7 +262,7 @@ bring "./subdir/structs_2.w" as otherExternalStructs;
 
 struct MyStruct {
   m1: externalStructs.MyStruct;
-  m2: otherExternalStructs.MyStruct;
+  m2: otherExternalStructs.MyStruct2;
```

Comment:
OK, yeah you're right - I looked into it and it's safe revert this change.

The reason changed it was because I noticed the code was organized so `subdir/structs.w` had a struct named `MyStruct` AND `subdir/structs_2.w` has a struct named `MyStruct`. Since the structs have the same name, it's not possible to `bring "./subdir"` safely (there would be a name collision). But `struct_from_json.test.w` doesn't actually import the entire directory, so that error doesn't come up.